topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperator.java

```diff
@@ -1202,6 +1203,7 @@ public ReconcileState(Set<TopicName> succeeded, Set<TopicName> undetermined, Map
             this.succeeded = succeeded;
             this.undetermined = undetermined;
             this.failed = failed;
+            this.ktList = emptyList();
         }
 
         public void addKafkaTopics(List<KafkaTopic> ktList) {
```

Comment:
Isn't this a setKafkaTopics?

Reply:
What do you mean? It is just patching Spotbugs violation of dereferencing field which was not initialized in the constructor.

Reply:
He's asking why the method is called `addKafkaTopics` and not `setKafkaTopics()`.